pkgs/applications/networking/mailreaders/thunderbird/default.nix

```diff
@@ -75,6 +75,7 @@ stdenv.mkDerivation rec {
     cd objdir
     echo '${stdenv.lib.concatMapStrings (s : "ac_add_options ${s}\n") configureFlags}' > .mozconfig
     echo 'ac_add_options --prefix="'"$out"'"' >> .mozconfig
+    echo 'ac_add_options --with-external-source-dir="'`realpath ..`'"' >> .mozconfig
```

Comment:
IIRC there was an attempt to just update version and hash, but that didn't build. (I don't even know the message, though.)